package.json

```diff
@@ -19,9 +19,13 @@
     "compression": "^1.5.2",
     "errorhandler": "^1.4.3",
     "express": "^4.13.3",
+    "express-minify": "^0.2.0",
     "express-sslify": "^1.0.1",
-    "jstransformer-uglify-js": "^1.1.1",
+    "fs-walk": "0.0.1",
```

Comment:
I accidentally branched off master, then cherry picked my commits to a branch off develop, must have missed that when cleaning up. I'll look at it sometime over the weekend.